tests/attributes/src/lib.rs

```diff
@@ -1,41 +1,9 @@
-use hax_lib_macros as hax;
-
-// dummy max value
-const u32_max: u32 = 90000;
-
-#[hax::requires(x > 10 && y > 10 && z > 10 && x + y + z < u32_max)]
```

Comment:
Why did you remove this code?

Reply:
Oops, that is a mistake (I have the bad habit of using this `attributes` test as a playground...)